docs/networking.md

```diff
@@ -0,0 +1,69 @@
+---
+title: Networking
+subtitle: Connecting ev3dev to the internet and other devices
+---
+
+If you want to access your ev3dev installation from a PC or install new software
+on your EV3, you'll need to connect to a network. Here are some resources to
+help you out.
+
+# Connecting to the internet
+
+You have a few options for connecting to the internet.
+
+### With a Wi-Fi dongle
+
+The simplest route is to purchase a Wi-Fi dongle and plug it into the EV3's USB
+port. Almost any USB Wi-Fi dongle which supports Linux will work, including the
+one that is supported by the official LEGO software. Once you have plugged in a
+dongle, you can use Brickman's "**_Wireless and Networks_ > _Wi-Fi_**" menu to
+connect to a network.
+
+### With a USB Ethernet adapter
+
+If you have a USB Ethernet adapter (or can buy one) which supports Linux, you
+can use it to access the network from your EV3. Just plug it into the brick and
+connect the adapter to an ethernet cable; no configuration is related.
```

Comment:
Damn autocorrect :laughing: I enabled a spellcheck extension for editing Markdown on the site, but apparently it got a bit overzealous after I mistyped something. Thanks for noticing!